style.css

```diff
@@ -0,0 +1,78 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'poppins';
+}
+body{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    background: #0e1538;
+}
+.container{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.container a{
+    position: relative;
+    width: 160px;
+    height: 60px;
+    display: inline-block;
+    background: #fff;
+    margin: 20px;
+}
+.container a::before,.container a::after{
+    content: '';
+    position: absolute;
+    inset: 0;
+    background: #111;
+    transition: 0.5s;
+}
+.container a span{
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: 1;
+    width: 100%;
+    height: 100%;
+    display: inline-block;
+    background: #0e1538;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-transform: uppercase;
+    font-size: 1.2em;
+    color: #fff;
+    letter-spacing: 2px;
+    border: 2px solid #040a29;
+    overflow: hidden;
```

Comment:
same here group the similar properties @faikp

Reply:
Yes Bhai , I'll do that changes👍🏻